apps/client/next.config.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
   reactStrictMode: true,
   transpilePackages: ['api-client'],
+  images: {
+    domains: ['image.tmdb.org'],
```

Comment:
since `domains` will be deprecated from Next 14 better to use [`remotePatterns`](https://nextjs.org/docs/app/api-reference/components/image#remotepatterns) :)